ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/graphql/GraphQLDataFetchers.java

```diff
@@ -230,13 +228,7 @@ public GraphQLDataFetchers(final Set<Capability> supportedCapabilities) {
       @SuppressWarnings("unchecked")
       final List<List<Bytes32>> topics = (List<List<Bytes32>>) filter.get("topics");
 
-      final List<List<LogTopic>> transformedTopics = new ArrayList<>();
```

Comment:
This will likely re-appear in this PR.  Migrating LogTopics to a DelegatingBytes should be done in a fresh PR as this one is huge enough.

Reply:
That's done now. Hopefully reduces the number of changes.